common/djangoapps/student/models.py

```diff
@@ -504,7 +504,7 @@ class Meta(object):
     allow_certificate = models.BooleanField(default=1)
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True, blank=True)
-    phone_regex = RegexValidator(regex=r'^\+?1?\d*$', message="Phone number can only contain numbers.")
+    phone_regex = RegexValidator(regex=r'^\(?\d{3}\)?-?\d{3}-?\d{4}$', message="Phone number can only contain numbers.")
```

Comment:
Nit: The message does not seem to match the regex

Reply:
🔧 'd